index.html

```diff
@@ -2435,37 +2435,58 @@ <h2>Cryptographic Suites</h2>
 verification of the data.
         </li>
         <li>
-          The specification MUST define a <dfn>suite selection algorithm</dfn> that accepts a [=JSON
-          object=] |proof| and uses the properties described in [[[#proofs]]] to return either
-          failure or a [=VC cryptographic suite=] [[Infra]] [=struct=]. This algorithm SHOULD be
-          registered in [[[VC-SPECS]]]. A
-          <dfn class="export">VC cryptographic suite</dfn> [=struct=] has the following
```

Comment:
You really should keep the exported definition, so that other specs can refer to the whole struct at once.

Reply:
Removing the export was a mistake, I'll fix it in the next revision. 

The one concern was calling it a "VC cryptographic suite" -- the only reason we're doing the Data Integrity work in the VCWG is due to politics. The Data Integrity technology is NOT supposed to be bound at all to the Verifiable Credential work (except that VCs are just ONE use of Data Integrity)... my concern in doing this (many years ago when the work was chartered) was that people would think that DI is tightly coupled to VCs, which is what certain actors wanted during the chartering process, but was never the intention of the work. All that to say "VC cryptographic suite" is wrong, "DI cryptographic suite" might be an ok compromise, but exporting it as "cryptographic suite" doesn't feel wrong either (since the xref would have to refer to the spec to import the language).

Just noting this here to see if you have any thoughts. I'll export the definition and adjust further if you have strong feelings one way or the other.

Reply:
Defining "cryptographic suite" is fine with me. I only picked the more specific term because I wasn't sure it's what the WG intended.

Reply:
I would go for "DI  Cryptographic Suite".

Reply:
FWIW, Bikeshed has xrefs that pull in everything, without needing the spec to be listed, so we'd want to resolve a collision on "cryptographic suite" if one happens, but resolving those isn't the end of the world. If the WG thinks the interface here will work for cryptographic suites outside of this ecosystem (like in https://w3c.github.io/webcrypto/), it'd be fine to claim the term, IMO. If the WG doesn't want to be that general, then picking the more specific term is better.

Reply:
> FWIW, Bikeshed has xrefs that pull in everything, without needing the spec to be listed

Ah, well, yes, that would be an issue. I'll go with the longer name for now. We could probably utilize the same concept for webcrypto, but that is far more work than this WG has signed up to do.

Reply:
I settled on `cryptographic suite instance` for now, and exported the definition.

Reply:
I understand it is a pain but, thinking a bit further, I do believe it is worth having a reference to DI in the name. If I look at the spec from a _purely Linked Data perspective_ using this spec for _any_ RDF Dataset, then there are some problems that these specs do not really solve (we would get back to the discussions we had with Henry Story, which could be settled properly for VC but not in general).

Solving those issues is not in the charter of this WG, so that is o.k., but that is why I believe a reference to DI would make sense.

Reply:
Done in 63631f56a77792ac48021dd472fb59cb2cf9db76.